src/main/java/com/alkemy/ong/model/request/UserDetailsRequest.java

```diff
@@ -0,0 +1,32 @@
+package com.alkemy.ong.model.request;
+
+import com.alkemy.ong.model.entity.Role;
+import java.util.List;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.Size;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class UserDetailsRequest {
+
+  private static final String EMPTY_FIELD = "the field cannot be empty";
+  private static final String BLANK_FIELD = "the field cannot be blank";
```

Comment:
Los mensajes de validaciones deben ser definidos para cada uno de los fields, sino no se sabe cual field no puede ser empty o null. Es decir el mensajes debería ser: "firstName cannot be blank" para el firstName y así sucesivamente para los demás atributos.